xdsl/dialects/arith.py

```diff
@@ -627,3 +580,47 @@ def get(operand1: Union[Operation, SSAValue],
         operand1 = SSAValue.get(operand1)
         return Minf.build(operands=[operand1, operand2],
                           result_types=[operand1.typ])
+
+Arith = Dialect([
```

Comment:
why moved here?

Reply:
You mean why I moved it to the end of the file? Since these are global variables, and Python files are executed line by line, having these at the start of the file makes my vscode complain as it the classes are not defined at this point. (I use Pylance, as well as the vscode Python extention, which comes with some linters/code checkers).